### PR TITLE
SOC-1869 Auth Dedicated Page Design Cleanup - improvements

### PR DESCRIPTION
**Pull request by rafalkalinski**
## Links

https://wikia-inc.atlassian.net/browse/SOC-1869

## Description

Styles for the join page were broken.

### CODE REVIEW

File: front/auth/app/styles/layout/_body.scss [hunk 1]
```diff
@@ -4,3 +4,10 @@ body {
 	justify-content: flex-start;
 	margin: auto;
 }
+
+.page-view {
+	display: flex;
+	flex-direction: column;
+	justify-content: flex-start;
```
Comment (Bloodmoon24) [inline front/auth/app/styles/layout/_body.scss]: please sort 'em ascending

**Comment by Bloodmoon24** [comment]
Besides that sorting, LGTM

File: front/auth/app/styles/theme/_splash.scss [hunk 2]
```diff
@@ -18,6 +18,12 @@
 	background-position: center bottom;
 }
 
+.join-page {
+	.footer-callout-emphasis {
+		font-weight: bold;
```
Comment (bkoval) [inline front/auth/app/styles/theme/_splash.scss]: Please, fix the order of properties.

**Comment by bkoval** [comment]
LGTM

**Pull request merged**